docs/source/faq.rst

```diff
@@ -107,8 +107,8 @@ Why I don't see any logs in the console?
 ----------------------------------------
 
 In version R19, Gunicorn doesn't log by default in the console.
-To watch the logs in the console you need to use the option ``--log-file=-``.
-In version R20, Gunicorn logs to the console by default again.
+To watch the logs in the console you need to use the option ``--error-logfile=-``.
```

Comment:
Also, it would be nice to mention that Gunicorn doesn't log access logs by default. They need to pass `--access-logfile=-` to make Gunicorn logs everything to the console.